scripts/python/testbedApi.py

```diff
@@ -0,0 +1,329 @@
+# Python testing utility classes and functions
```

Comment:
These are quick and dirty tests i created so that I can run them on Macs and Linux. I wouldn't read much into it at the moment. We can have bigger discussion on what to do about running system tests on Macs going forward.

Erik, I didn't know python till last week. Learning it was surprisingly easy since syntax is somewhat similar to Golang. Now, i'm feeling like may be I should try ruby too :)